types/CustomerCashBalanceTransactions.d.ts

```diff
@@ -126,6 +126,7 @@ declare module 'stripe' {
       type Type =
         | 'applied_to_payment'
         | 'funded'
+        | 'funding_reversed'
```

Comment:
Looks like this wasn't included in the Changelog?

Reply:
Hmm, weird